synapse/cores/common.py

```diff
@@ -247,15 +284,28 @@ def _saveCoreModel(self):
                     fullprop = '%s:%s' % (name,prop)
                     self.formTufoByProp('syn:prop',fullprop,**pnfo)
 
-    def _loadCoreModel(self):
-        for item in self.getTufosByProp('syn:type'):
-            self._initTypeTufo(item)
+    def addDataModel(self, modname, modl):
+        '''
+        Store all types/forms/props from the given data model in the cortex.
 
-        for item in self.getTufosByProp('syn:form'):
-            self._initFormTufo(item)
+        Example:
 
-        for item in self.getTufosByProp('syn:prop'):
-            self._initPropTufo(item)
```

Comment:
well git's diff algorithm is totally failing here. this function is not touched. focus on `addDataModel` and `addDataModels`.